### PR TITLE
New BEE Book in vending machines and Apiarist pockets

### PR DESCRIPTION
**Pull request by keiyakins**
## About the PR 

Adds the Bee Exposition Extravaganza - BEE Book for short - to Books4u. The sprite is admittedly a little bit programmer arty, but I think it's Good Enough unless an actual artist wants to redo it (in which case *bee* my guest!)

## Why's this needed? 

There's really no good in-game guide to bees. Also, I was bored and felt like writing it.

## Changelog

```changelog
(u)Keiya
(*)New "Bee Exposition Extravaganza" book, now available at Books4U vendors system-wide! Your local apiculturist might have a copy, too. 
```

### CODE REVIEW

**Comment by TobleroneSwordfish** [comment]
🐝

**Comment by codeJester27** [comment]
Should the Apiculturist/Apiarist spawn with the book?

**Comment by keiyakins** [comment]
Good idea. Their inventory is packed when they spawn, but I can replace the pen since there's one in the PDA.

File: strings/books/bee_book.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<p>Woah... these Bees are like, far out man... their eyes flicker and when they hug you it buzzes real good. Exposure to certain substances as an egg or larva causes Bees to become this when they grow up.</p>
+<h2 id="chromatic-space-bee">Chromatic Space-Bee</h2>
+<p>These Bees change between different colors constantly! Very popular among hardcore gamers. Basically never seen without Psychadelic Space-Bees in the area.</p>
+<h2 id="zombees">Zombees</h2>
+<p>These Bees look unwell... but they'll produce honey for you just the same, if you give them lots of love!</p>
+<h1 id="chapter-three-specific-bees">Chapter Three: Specific Bees</h1>
+<h2 id="heisenbee">Heisenbee</h2>
+<p>The beloved pet of the Research Director of NanoTrasen Space Station 13. Tends to wear extremely stylish hats.</p>
+<h2 id="bombini">Bombini</h2>
+<p>This li'l guy deals in all sorts of Bee-related goods. Definitely someone you want to Bee-friend!</p>
```
Comment (TTerc) [inline strings/books/bee_book.txt:28]: Minor nitpick, Bombini is a she. Overall I enjoyed reading the book, very cute and informative

Comment (keiyakins) [reply strings/books/bee_book.txt]: Oh, huh, I never really thought about it, but that makes sense. I'll fix it right away.

**Review by TTerc — APPROVED** [review]
I enjoy

**Pull request merged**